angularSTEP/src/environments/environment.ts

```diff
@@ -3,7 +3,17 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
```

Comment:
Try not to commit this file. I did the same thing but am trying to just not add it to be tracked. I can change the key again.

Reply:
I'll double check that I don't add it to the commit in the future

Reply:
Can this file at least be removed from this PR before its merged?